README.md

```diff
@@ -93,6 +93,30 @@ First, update the repository (assuming you have `master` checked out):
 Look carefully for errors with `git pull`, and reset locally modified files if
 necessary.
 
+## Upgrading Instructions 
+
+*__Only necessary while upgrading from upgrading from v1.3.2.0.0.0.0.0.0.1 or below.__*
+
+1. Drop the old dcrdata database and create a new empty dcrdata database.
```

Comment:
In db/dcrpg/tables.go, please bump `tableMajor` to 3 for this change.

Reply:
Noted